layouts/shortcodes/contributors.html

```diff
@@ -0,0 +1,31 @@
+
+<table class="tableblock stretch">
+    <thead>
+        <tr>
+            <th>Avatar</th>
+            <th>Name</th>
+            <th>Organization</th>
+        </tr>
+    </thead>
+    <tbody>
+        {{ range  $page_count := (seq 10) }}
```

Comment:
Option 1: comment so we know what this is about
```suggestion
        {{/* TODO we can't access HTTP headers and to fetch all contributors we need to load multiple pages, when Camel has over 1000 contributors the 10 pages we have here will not be enough */}}
        {{ range  $page_count := (seq 10) }}
```

Option 2: do a `range` over much bigger sequence and don't invoke `getJSON` if last invocation returned 100 contributors, something like:
```
{{/* we're looping over 100 numbers in order to fetch paged data from GitHub API, we're invoking GitHub API only if we received all results per page (100) from the last page */}}
{{ $pages := seq 100 }}¬
{{ $last_contributors := 1 }}¬
{{ range $page, $tmp := $pages }}¬
    {{ if eq $last_contributors 100 }}¬
        {{ $git_contributors := getJSON "https://api.github.com/repos/apache/camel/contributors?per_page=100&anon=1&page=" ($page | string) }}¬
        {{ $last_contributors = len $git_contributors }}¬
...
    {{ end }}¬
{{ end }}¬

```

Reply:
This outputs a blank table[img1] but when `$last_contributors := 100` it works but with an API limit error[error]
![img1](https://user-images.githubusercontent.com/49795064/78472594-186ba780-7732-11ea-83c7-ec0217b88dbe.jpeg)


![error](https://user-images.githubusercontent.com/49795064/78472604-333e1c00-7732-11ea-9431-3177d97f6cdd.PNG)